### ISSUE TITLE
Failed to build from source with Qt 5.15

### ISSUE THREAD

**Issue by hexchain** [otfried/ipe#321]
It seems that from Qt 5.15 one needs to have `<QPainterPath>` included to use that object:

```
ipecanvas_qt.cpp:61:16: error: field ‘iPP’ has incomplete type ‘QPainterPath’
   61 |   QPainterPath iPP;
      |                ^~~
In file included from /usr/include/qt/QtGui/qbrush.h:49,
                 from /usr/include/qt/QtGui/qpalette.h:46,
                 from /usr/include/qt/QtWidgets/qwidget.h:48,
                 from /usr/include/qt/QtWidgets/QWidget:1,
                 from ipecanvas_qt.h:37,
                 from ipecanvas_qt.cpp:31:
/usr/include/qt/QtGui/qmatrix.h:54:7: note: forward declaration of ‘class QPainterPath’
   54 | class QPainterPath;
      |       ^~~~~~~~~~~~
ipecanvas_qt.cpp: In member function ‘virtual void IpeQtPainter::doNewPath()’:
ipecanvas_qt.cpp:77:22: error: invalid use of incomplete type ‘class QPainterPath’
   77 |   iPP = QPainterPath();
      |                      ^
In file included from /usr/include/qt/QtGui/qbrush.h:49,
                 from /usr/include/qt/QtGui/qpalette.h:46,
                 from /usr/include/qt/QtWidgets/qwidget.h:48,
                 from /usr/include/qt/QtWidgets/QWidget:1,
                 from ipecanvas_qt.h:37,
                 from ipecanvas_qt.cpp:31:
```

There are multiple similar errors from other places.

**Comment by otfried** [comment]
You seem to be on the cutting edge.   Even Ubuntu 20.04 still uses Qt 5.12.

Fixed in 7.2.19.

**Comment by hexchain** [comment]
Yes, I am using Arch Linux with `kde-unstable` repo (for Plasma 5.19), the Qt version is 5.15.0rc2.

Thanks for fixing!
